appveyor.yml

```diff
@@ -0,0 +1,37 @@
+os: Visual Studio 2015
+
+artifacts:
+  - path: godot\bin\*.exe
+
+environment:
+  PYTHON: "C:\\Python27"
+  BUILD_REVISION: official
+  matrix:
+    - VS: C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat
+      TOOLS: yes
+      ARCH: amd64
+    - VS: C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat
+      TOOLS: yes
+      ARCH: x86
+    - VS: C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat
+      TOOLS: no
+      ARCH: amd64
+    - VS: C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat
+      TOOLS: no
+      ARCH: x86
+
+install:
+  - SET "PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
+  - pip install --egg scons  # it will fail on AppVeyor without --egg flag
```

Comment:
error - https://ci.appveyor.com/project/Marqin/godot/build/1.0.2
the same after upgrading pip - https://ci.appveyor.com/project/Marqin/godot/build/1.0.3